source/isaaclab/isaaclab/sensors/imu/imu.py

```diff
@@ -27,10 +27,12 @@
 class Imu(SensorBase):
     """The Inertia Measurement Unit (IMU) sensor.
 
-    The sensor can be attached to any :class:`RigidObject` or :class:`Articulation` in the scene. The sensor provides complete state information.
-    The sensor is primarily used to provide the linear acceleration and angular velocity of the object in the body frame. The sensor also provides
-    the position and orientation of the object in the world frame and the angular acceleration and linear velocity in the body frame. The extra
-    data outputs are useful for simulating with or comparing against "perfect" state estimation.
+    The sensor can be attached to any prim path and produces body-frame linear acceleration and angular velocity,
```

Comment:
What happens if there isn't any rigid ancestor up the tree?
```suggestion
    The sensor can be attached to any prim path with a rigid ancestor in its tree and produces body-frame linear acceleration and angular velocity,
```